### PR TITLE
Fix #281: "Mark all read" functionality is missing

### LINKED ISSUE #133: mark all alerts as read
it'd be nice if we could have an option to mark all of the alerts as being read without having to visit each alert and/or delete all of them.

### LINKED ISSUE #281: "Mark all read" functionality is missing

### PR DESCRIPTION
**Pull request by lairdshaw**
Resolves #281

Resolves #133

### CODE REVIEW

**Comment by Eldenroot** [comment]
@lairdshaw - could you please update the code? I would like to update the plugin on my board to the latest GitHub :) Thank you ;)

**Comment by lairdshaw** [comment]
Patience, my friend. I am whittling down my effective to-do list slowly but surely. Rest assured that this task remains on it.

File: alerts.php [hunk 1]
```diff
@@ -230,6 +233,47 @@ function myalerts_delete_alert($mybb, $db, $lang)
...
+	$alertIds = array();
+
+	if (!empty($alertsList) && is_array($alertsList)) {
+		foreach ($alertsList as $alertObject) {
+			$alert = parse_alert($alertObject);
+			$alertIds[] = $alert['id'];
+		}
+	}
+
+	MybbStuff_MyAlerts_AlertManager::getInstance()->markRead($alertIds);
```
[the remark this answers is not in the document]
Comment (lairdshaw) [reply alerts.php]: Resolved in https://github.com/MyBBStuff/MyAlerts/pull/282/commits/c0b585b98b5c506c75a6e70d7a7a8ff704f3fe9a

I think this PR's ready to merge.

**Comment by Eldenroot** [comment]
This now should be complete, added to my testing build and put on my board for testing

**Pull request merged**